hypha/apply/funds/templates/funds/includes/review_sidebar.html

```diff
@@ -19,8 +19,13 @@
 
 <ul class="mt-4 reviews-sidebar" id="id_reviewers" x-data='{showHiddenReviewers: false}'>
     {% if not staff_reviewers_exist %}
-        <li class="reviews-sidebar__no-reviews">{% trans "No staff reviewers yet" %}</li>
-        <hr class="reviews-sidebar__split">
+        {% if request.user.is_org_faculty %}
+            <li class="reviews-sidebar__no-reviews">{% trans "No staff reviewers yet" %}</li>
+            <hr class="reviews-sidebar__split">
+        {% else %}
+            <li class="reviews-sidebar__no-reviews">{% trans "No public reviews yet" %}</li>
```

Comment:
Do we want to let them know that there might be some private reviews otherwise we could say "No reviews yet"?

Reply:
@wes-otf I think this is a good question. Should we just hide reviews completely, or should we say that there are x numbers of them but you can't view them.